src/Symfony/Component/Yaml/Inline.php

```diff
@@ -249,7 +249,7 @@ public static function parseScalar($scalar, $delimiters = null, $stringDelimiter
                 $output = $match[1];
                 $i += strlen($output);
             } else {
-                throw new ParseException(sprintf('Malformed inline YAML string (%s).', $scalar));
+                throw new ParseException(sprintf('Malformed inline YAML string: %s.', $scalar));
```

Comment:
yeah, messages without brackets were changed in the past precisely for the reason I gave. Looks like messages added in newer branches were not updated after the branch merge